db/migrations/versions/2024_05_01_1700-4a0c70f02342_added_submitting_account_id_submitting_.py

```diff
@@ -0,0 +1,34 @@
+"""Added submitting_account_id & submitting_user_email fields to Submission model.
```

Comment:
Can you rename this migration file to follow the naming convention of the others? You can then update the revision ID referenced both in the docstring and as the revision var, to match the filename (see previous ones for examples)

Reply:
Migration filename has been changed to match naming convention.